.travis.yml

```diff
@@ -15,10 +15,10 @@ install:
   - sudo ln -s /tmp/nextflow/nextflow /usr/local/bin/nextflow
   # Set up nf-core and test/coverage modules
   - cd ${TRAVIS_BUILD_DIR}
-  - sudo pip install .
-  - sudo pip install codecov nose nose2
+  - pip install .
+  - pip install codecov nose pytest pytest-cov
```

Comment:
Do we still need `nose`?

Reply:
only for the `@raises` annotation so far, I can search for a replacement

Reply:
Done